templates/todo/api/csharp-cosmos-sql/TodoEndPointsExt.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+using SimpleTodo.Api;
+
+namespace Todo.Api
+{
+    public static class TodoEndpointsExt
+    {
+        public static void MapTodoEndpoints(this WebApplication app)
+        {
+
+            app.MapGet("/lists", async (ListsRepository _repository, [FromQuery] int? skip = null, [FromQuery] int? batchSize = null) =>
+            {
+                return TypedResults.Ok(await _repository.GetListsAsync(skip, batchSize));
+            })
+                .Produces(200)
+                .WithOpenApi();
+
+
+            app.MapPost("/lists", async (ListsRepository _repository, [FromBody] CreateUpdateTodoList list) =>
+            {
+                var todoList = new TodoList(list.name)
+                {
+                    Description = list.description
+                };
+
+                await _repository.AddListAsync(todoList);
+
+                return Results.Created($"/lists/{todoList.Id}", todoList);

```

Comment:
Recommend using `TypedResults` here. Similarly, you won't need `Produces(201)` if you use the `TypedResults` variant.